### ISSUE TITLE
Add support for large screen devices - Desktop, Web, Tablets

### ISSUE THREAD

**Issue by AlejandraPedroza** [msasikanth/twine#732]
Hello,

I'm a Kotlin technical writer at JetBrains, currently working on updating the [Kotlin Multiplatform samples page](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-samples.html). This project is listed there, and as part of the update, I am mapping the samples to the platforms they support. I noticed that this project does not yet support Compose for Web.

Given our goal of providing users with the best multiplatform experience, I wonder if there's an opportunity to update this project to include support for the web platform as well.

Thank you in advance for considering this!

Best regards,
Alejandra.

**Comment by msasikanth** [comment]
Hi Alejandra, at the moment I was only considering large-screen devices & desktops for more options. I was waiting for the adaptive navigation suite to be able to make it easier for me to build those. Now that they are available, I will soon visit the idea of desktop/large screen devices.

TBH I haven't consider web as a platform to support, since that requires hosting it and building backend and things like that which I have little experience with. But I can consider a self hosting idea that be open source. But my primary consideration for another platform is just desktop.

**Comment by AlejandraPedroza** [comment]
Hi!
Thanks for your answer. That would be great. Please let's stay in touch for the platforms you add so that we update the samples page. 
Best regards,
Alejandra.

**Comment by scramiro** [comment]
I'd love to read on tablets. I also miss the offline button, but syncing and reading through android platforms was the thing to go for other apps. It would be a great idea to add a feature timeline in the app to encourage people to subscribe (haven't seen any update for a while so I wonder about developers support)

**Comment by msasikanth** [comment]
Yeah, large screen support is still planned and I want to make it. I just have to find some time to work on it. Hopefully soon 🤞

**Comment by msasikanth** [comment]
Hi @AlejandraPedroza , I have updated the project to suppor desktop target using Compose Desktop. Libraries have also been updated, and I have included the latest tech stack in my updated README
